home/README.md

```diff
@@ -1,2 +1,41 @@
-[![Build Status](https://travis-ci.org/ethcore/parity-dapps-builtins-rs.svg?branch=master)](https://travis-ci.org/ethcore/parity-dapps-builtins-rs)
-# Parity Built-in Dapps
+# Parity Home Page & TopBar
+
+Parity Home Page (DApps list) and TopBar.
+
+[Back to parity-ui](../README.md)
+
+## Development
+
+```bash
+ $ cd parity-ui/home/web
+ $ npm install
+ $ npm start # Starts development server
+ $ xdg-open http://localhost:3010 || open http://localhost:3010
+```
+
+Make sure to have Parity running.
+
+## Pre-compiling JS files
+
+After you finish changing the source code you need to also commit pre-compiled JS files, otherwise the build will fail.
+
+```bash
+ $ cd parity-ui/home/web
+ $ rm -rf node_modules                  # Make sure to start with fresh dependencies
+ $ rm -rf ../../components/node_modules
+ $ npm install --ignore-scripts         # Install dependencies (but don't link components)
+ $ NODE_ENV=production npm run build
```

Comment:
I don't see a point in having this script. It's always safer to recompile everything with `web.sh clean && web.sh build`. I think it's very unlikely that we will run builds manually in near future anyway.